src/keybindings.c

```diff
@@ -824,6 +824,15 @@ static void apply_kb_accel(GeanyKeyGroup *group, GeanyKeyBinding *kb, gpointer u
 }
 
 
+/** Reloads keybinding settings from configuration file. Normally plugins do
+ * not need to call this function as it is called automatically when a the
+ * plugin is activated. However, plugins which need to create keybindings
+ * dynamically and reload them when needed should call this function after
+ * all keybindings have been updated with plugin_set_key_group() and
+ * keybindings_set_item() calls - this makes sure that the corresponding user
+ * keybinding shortcuts are applied.
+ * @since 1.31. */
```

Comment:
Recently we used "@since X.YY (API ZZZ)" (with api version, without trailing dot). The API version tends to be more interesting for plugin developers. Please change this accordingly.